pallets/ocex/src/lib.rs

```diff
@@ -361,6 +363,8 @@ pub mod pallet {
 			max_order_qty: BalanceOf<T>,
 			price_tick_size: BalanceOf<T>,
 			qty_step_size: BalanceOf<T>,
+			base_asset_precision: u8,
```

Comment:
We don't need this input from the user, we can calculate it from scale()

Reply:
Updated ✅